src/Microsoft.Identity.Web/TokenCacheProviders/CacheSerializerHints.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+
+namespace Microsoft.Identity.Web.TokenCacheProviders
+{
+    /// <summary>
+    /// todo.
+    /// </summary>
+    public class CacheSerializerHints
+    {
+        /// <summary>
+        /// Todo.
```

Comment:
CancellationToken enabling cooperative cancellation between threads, thread pool, or Task objects. This is used
when calling the token acquisition methods, which then pass-them in to the token cache serializers.